lib/dotcom_web/live/trip_planner.ex

```diff
@@ -71,19 +73,53 @@ defmodule DotcomWeb.Live.TripPlanner do
         </.async_result>
       </section>
 
-      <.live_component
-        module={TripPlannerResultsSection}
-        id="trip-planner-results"
+      <.trip_planner_results_section
         results={@results}
         error={@error}
         map_config={@map_config}
         from={@from}
         to={@to}
+        itinerary_selection={@itinerary_selection}
       />
     </div>
     """
   end
 
+  @impl true
+  def handle_event("set_itinerary_group_index", %{"index" => index_str}, socket) do
```

Comment:
One of the reasons I originally made everything live components was because of these event names; it felt a bit odd to have the event names spread around the way they are - the handler is here, but the thing that generates the event is tucked away in `trip_planner_results_section` (or for the `set_itinerary_index` event, tucked away in `itinerary_detail`).

Is that okay? Do we want to pass the event names around as `attr`s? Some other solution?

Reply:
Realistically, I could see the answer to this being a follow-up, rather than trying to "fix" it in this PR, but I am interested in what our overarching vision for this type of question is.